modules/zypper/init.zsh

```diff
@@ -0,0 +1,31 @@
+#
+# Defines zypper aliases.
+#
+# Authors:
+#   Simon <contact@saimon.org>
+#   Sorin Ionescu <sorin.ionescu@gmail.com>
+#   Andrea Ceccarelli <gltch2003@gmail.com>
+#
+
+# Return if requirements are not found.
+if (( ! $+commands[zypper] )); then
+  return 1
+fi
+
+#
+# Aliases
+#
+
+alias zypc='sudo zypper clean -a'                     # Cleans the cache.
+alias zyph='zypper history'                           # Displays history.
+alias zypi='sudo zypper install'                      # Installs package(s).
+alias zypl='zypper clean'                             # Clean packages cache.
+alias zypq='zypper info'                              # Displays package information.
+alias zypr='sudo zypper remove'                       # Removes package(s).
+alias zyps='zypper search'                            # Searches for a package.
+alias zypv='sudo zypper ve'                           # Verifies dependencies.
```

Comment:
I'd prefer the full versions of these commands (`verify` rather than `ve`) for clarity